bazel/emscripten_toolchain/wasm_cc_binary.bzl

```diff
@@ -119,28 +121,35 @@ def _wasm_cc_binary_impl(ctx):
         if not valid_extname:
             fail("Invalid output '{}'. Allowed extnames: {}".format(output.basename, ", ".join(_ALLOW_OUTPUT_EXTNAMES)))
 
+    inputs = ctx.files.cc_target
+    args = ctx.actions.args()
     args.add_all("--archive", ctx.files.cc_target)
-    args.add_joined("--outputs", ctx.outputs.outputs, join_with = ",")
+    args.add_joined("--outputs", outputs, join_with = ",")
+
+    if dwp_file:
+        args.add("--dwp_file", dwp_file)
```

Comment:
How about `--dwp-file`?   At least for emscripten and emsdk we use the hyphen rather than underscore in the option name.

Reply:
I was guided by the `--allow_empty_outputs` option in the [`bazel/emscripten_toolchain/wasm_binary.py`](bazel/emscripten_toolchain/wasm_binary.py) file.

Reply:
@sbc100 If you wish, I can change both.

Reply:
No worries, its all internal anyway, so keeping the local convention seems fine.